src/ui/universeview.cpp

```diff
@@ -145,201 +133,96 @@ void UniverseView::listenerStarted(int universe)
-            ui->twSources->item(row,COL_PATHWAY_SECURE)->setText(tr("No"));
-            ui->twSources->item(row,COL_PATHWAY_SECURE)->setBackground(colourBad);
-        } else {
-            ui->twSources->item(row,COL_PATHWAY_SECURE)->setText(tr("N/A"));
-            ui->twSources->item(row,COL_PATHWAY_SECURE)->setBackground(Qt::transparent);
-        }
-    }
-
     // Update select address details
     if (m_selectedAddress != NO_SELECTED_ADDRESS)
         selectedAddressChanged(m_selectedAddress);
 }
 
-void UniverseView::sourceOnline(sACNSource *source)
-{
-    if (!m_listener)
-        return;
-
-    // Display sources that only transmit 0xdd?
-    if (!m_displayDDOnlySource && !source->doing_dmx)
-        return;
-
-    int row = ui->twSources->rowCount();
-    ui->twSources->setRowCount(row+1);
-    m_sourceToTableRow[source] = row;
-
-    for (auto col = 0; col < COL_END; col++)
-        ui->twSources->setItem(row, col, new QTableWidgetItem() );
-
-    // Seq errors, with reset
-    {
-        QWidget* pWidget = new QWidget();
-        QHBoxLayout* pLayout = new QHBoxLayout(pWidget);
-        pLayout->setAlignment(Qt::AlignCenter);
-        pLayout->setContentsMargins(3,0,0,0);
-
-        // Count
-        QLabel* lbl_seq = new QLabel();
-        lbl_seq->setText(QString::number(0));
-        pLayout->addWidget(lbl_seq);
-
-        // Reset Button
-        QPushButton* btn_seq = new QPushButton();
-        btn_seq->setIcon(QIcon(":/icons/clear.png"));
-        btn_seq->setFlat(true);
-        pLayout->addWidget(btn_seq);
-
-        // Connect button
-        connect(btn_seq, &QPushButton::clicked, this, [=]() {
-            source->resetSeqErr();
-            this->sourceChanged(source);
-        });
-
-        // Display!
-        pWidget->setLayout(pLayout);
-        ui->twSources->setCellWidget(row,COL_SEQ_ERR, pWidget);
-    }
-
-    // Jump counter, with reset
-    {
-        QWidget* pWidget = new QWidget();
-        QHBoxLayout* pLayout = new QHBoxLayout(pWidget);
-        pLayout->setAlignment(Qt::AlignCenter);
-        pLayout->setContentsMargins(3,0,0,0);
-
-        // Count
-        QLabel* lbl_jumps = new QLabel();
-        lbl_jumps->setText(QString::number(0));
-        pLayout->addWidget(lbl_jumps);
-
-        // Reset Button
-        QPushButton* btn_jumps = new QPushButton();
-        btn_jumps->setIcon(QIcon(":/icons/clear.png"));
-        btn_jumps->setFlat(true);
-        pLayout->addWidget(btn_jumps);
-
-        // Connect button
-        connect(btn_jumps, &QPushButton::clicked, this, [=]() {
-            source->resetJumps();
-            this->sourceChanged(source);
-        });
-
-        pWidget->setLayout(pLayout);
-        ui->twSources->setCellWidget(row,COL_JUMPS, pWidget);
-    }
-
-
-    sourceChanged(source);
-}
-
-void UniverseView::sourceOffline(sACNSource *source)
-{
-    if (!m_listener)
-        return;
-    sourceChanged(source);
-}
+//void UniverseView::sourceOnline(sACNSource *source)
```

Comment:
Large block of commented out code - can we just remove it?

Reply:
I'd missed that entirely. Thanks!